package.json

```diff
@@ -4,43 +4,125 @@
   "description": "",
   "main": "src/index.js",
   "scripts": {
-    "build": "wp-scripts build && npm run theme",
-    "format:js": "wp-scripts format-js",
-    "lint:js": "wp-scripts lint-js",
-    "packages-update": "wp-scripts packages-update",
-    "start": "wp-scripts start",
+    "build": "wds-build && npm run theme",
+    "format": "wds-format-js && wds-format-styles && npm run format:prettier",
+    "format:prettier": "[[ -f .prettierignore ]] && ./node_modules/.bin/prettier ./ --write",
+    "lint": "run-s lint:js lint:styles lint:php",
+    "lint:js": "wds-format-js && wds-lint-js",
+    "lint:php": "composer install --quiet && composer run lint",
+    "lint:styles": "wds-format-styles && wds-lint-style",
+    "packages-update": "wds-packages-update",
     "postinstall": "npm run build",
-    "lint": "npm run lint:js && npm run lint:style",
-    "lint:style": "wp-scripts lint-style",
-    "theme": "run-s pot icons",
-    "pot": "wp i18n make-pot . build/languages/_s.pot --exclude=node_modules,vendor,build",
-    "icons": "svgstore build/images/icons/*.svg --inline -o build/images/icons/sprite.svg"
+    "preinstall": "$(echo node --version) | grep -sq $(cat .nvmrc) || ( echo \"\\n\\n\\n\\t!!! Please use Node version $(cat .nvmrc), you are using version $(node --version), some things may fail to work. !!!\\n\\n\\n\\n\" )",
+    "start": "wds-start",
+    "theme": "run-s theme:pot theme:icons",
+    "theme:icons": "svgstore build/images/icons/*.svg --inline -o build/images/icons/sprite.svg",
+    "theme:pot": "wp i18n make-pot . build/languages/_s.pot --exclude=node_modules,vendor,build",
+    "test": "npm install && run-s lint build && [[ -d build ]] && [[ -d src ]] && [[ -d inc ]] && [[ -d template-parts ]] && [[ -f style.css ]] && [[ -f webpack.config.js ]]"
```

Comment:
Yes, this is just a quick way to test the health of the package.....

- See if install works
- Run lint, see if that works
- Run build, see if that works

If all that worked we should have

- A build folder

Also check for some other files, that if were missing might be terrible:

- A src folder
- A inc folder
- A template-parts folder
- A style.css file which makes this a theme
- A webpack config so you can do cool 💩